### PR TITLE
[Merged by Bors] - feat(RingTheory): lemmas about annihilator

### PR DESCRIPTION
**Pull request by erdOne**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RingTheory/Ideal/Maps.lean [hunk 1]
```diff
@@ -722,6 +722,20 @@ theorem LinearEquiv.annihilator_eq (e : M ≃ₗ[R] M') :
...
+
+lemma Module.annihilator_eq_bot {R M} [Ring R] [AddCommGroup M] [Module R M] :
+    Module.annihilator R M = ⊥ ↔ FaithfulSMul R M := by
+  rw [← le_bot_iff]
+  refine ⟨fun H ↦ ⟨fun {r s} H' ↦ ?_⟩, fun ⟨H⟩ {a} ha ↦ ?_⟩
+  · rw [← sub_eq_zero]
+    exact H (Module.mem_annihilator (r := r - s).mpr
+      (by simp only [sub_smul, H', sub_self, implies_true]))
+  · exact @H a 0 (by simp [Module.mem_annihilator.mp ha])
 namespace Submodule
```
Comment (alreadydone) [inline Mathlib/RingTheory/Ideal/Maps.lean:739]:
```suggestion
  · exact @H a 0 (by simp [Module.mem_annihilator.mp ha])

namespace Submodule
```

**Review by alreadydone — APPROVED** [review]
LGTM, but why is this into the branch erd1/isInducing_of_nhds_one? There's no dependency either way.

**Comment by erdOne** [comment]
Oops, fixed (hopefully)

**Review by alreadydone — APPROVED** [review]
Thanks!
maintainer merge

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

**Pull request closed without merging**